src/actors/containerization/services/wordpress/wordpress_generator/wordpress_generator.py

```diff
@@ -0,0 +1,57 @@
+import json
```

Comment:
please run ```flake8 --max-line-length=120 .```

Reply:
Why 120 and not the default 80 (79)?

Reply:
At the beginning of this project we decided to use 120 characters, it's much more readable.

My personal opinion: PEP8 with 80 characters it's outdated standard, 120 is optimal, it's not like you have to deal with small resolutions nova days.

Reply:
I understand.

Personally, I'm unable to fit two text editor windows side-to-side and see 80+ lines unwrapped :-(